### DISCUSSION TITLE
QfieldCloud: Admin of organization and creator of projects. Problematic safety issues and problems with access for the creator of projects.

### DISCUSSION THREAD

**Discussion by VilleBio** [opengisch/QField#3918] [Q&A]
An admin of the organization can delete the organization. 

The creator of a project, that is then transfered to the organization, can then not access the project unless the creator is an admin of the organization or is invited by another admin. 

This is problematic because there is an option for admins to delete the organization (this could happen because of a misstake in a stressfull situation, or let's say because an admin might do it on purpose). 

As a result we as a big company would ideally want every employee to be able to upload a project to the organization and then access it without being an admin. This is not how it works at the moment. If the company grants adminstatus to counteract this the problem then is that we get the safety issue of that everyone could delete the organization. 

If only one admin rules the organization then this admin will have spend a lot of time just giving access to projects to every member within the organization.

How do we solve this problem?
(A suggestion would be to have a third type of user along with "admin" and "member": "semi-admins" which are authorized for doing limited actions (accessing projects/inviting collaborators). Or, simply make it so that every member can access all projects, especially the one the member him-/herself created.  I'm sure there are even better ways to solve this, but this is what I could think of as solutions while I'm presenting this issue).

Best regards, 
Vilhelm

**Comment by suricactus** [comment]
@VilleBio interesting case and well defined issue. 

A quick solution for you would be to add all org members to a team and add that team as a single collaborator to all your projects.

Any of your suggestions need QFieldCloud code modification. Currently we do not have plans to change any of the QFieldCloud role management and permissions. Feel free to drop us an email at info@opengis.ch if you are interested in getting your suggestions implemented.
